test/val/val_id_test.cpp

```diff
@@ -148,6 +153,7 @@ TEST_F(ValidateIdWithMessage, OpLineFileBad) {
 %4 = OpVariable %3 Input)";
   CompileSuccessfully(spirv.c_str());
   EXPECT_EQ(SPV_ERROR_INVALID_ID, ValidateInstructions());
+  EXPECT_THAT(getDiagnosticString(), HasSubstr("ID 1 has not been defined"));
```

Comment:
This is not the right message.  The message is an SSA problem.
But the actual bug is at line 150, since %2 is supposed to reference an OpString that names a file.

I don't even know why this message would be defined because I don't think anything references %1.